modules/ROOT/pages/audit-logging.adoc

```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+Occasionally, there are `Anypoint Staff` log entries. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
+`Anypoint Staff` entries are created for when these events occur:
+
+* An internal or system processes is performed
+* The SalesOps staff changes entitlements for your organization 
+* An action is taken using a bearer token associated with a MuleSoft employee account (for example, during troubleshooting)
```

Comment:
I know we got this directly from support, but the way it's worded is very much tied to how support thinks and troubleshoots the platform. Bearer tokens here don't really mean much. We should say something like "An action is taken by a MuleSoft employee on your behalf for other purposes, such as troubleshooting". Note the "other", since it's really a less common variation of the above bullet